src/primo/browzine-primo-adapter.js

```diff
@@ -871,6 +871,14 @@ browzine.primo = (function() {
     return validation;
   };
 
+  function shouldAvoidUnpaywall(response) {
+    if (response.hasOwnProperty('meta') && response.meta.hasOwnProperty('avoidUnpaywall')) {
```

Comment:
Shouldn't this use the value of `response.meta.avoidUnpaywall` rather than just check for the presence of a value?  What if someone made a CMS change that make the response have an `avoidUnpaywall` value of false for some reason?  With this code, a value of false would make the discovery service adapter avoid making an unpaywall call.

Reply:
I'm not sure I follow. This function returns the value of avoidUnpaywall and then on line 999 we interpret that value so that if it's true, we won't make an unpaywall call, but if its false we will. So in the scenario you mentioned, if avoidUnpaywall = false, then we'd make an unpaywall call.

Reply:
Oh nevermind, I was just thrown off by the use of `hasOwnProperty`.  This whole function could be replaced with 
```
return !!(response.meta && response.meta.avoidUnpaywall);
```
Either way looks good to me now.

Reply:
I find the way i've written this part to be easier to understand at a glance, so if that's okay with you then are we good to merge?

Reply:
Yep this is fine to merge.